utils/db.py

```diff
@@ -399,8 +403,8 @@ def set_yaml_config(config_name, data_dict, hostname=None):
 
 
 @contextmanager
-def database_on_server(hostname, **kwargs):
-    db_obj = Db(hostname=hostname, **kwargs)
+def database_on_server(appliance, **kwargs):
```

Comment:
I don't believe this is used anywhere. Search comes back with nothing. I say get rid of this completely.